src/InputField.js

```diff
@@ -9,72 +9,144 @@ import Label from './Label'
 import Input from './Input'
 import theme from './theme'
```

Comment:
I think that we should not import `theme` in the component unless you would like to use it as a default prop as what @jes708 did on some other components. However, there are sometimes a lot of gotchas in this method if the component has a lot of other nested components that depend on theme during testing. The other alternative is to wrap the test with a `ThemeProvider`. And I see that you are not using styled-component here, so you are good to go for now.